src/unstract/adapters/llm/ollama/src/static/json_schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "title": "Ollama AI LLM",
+  "type": "object",
+  "required": [
+    "base_url",
+    "model"
+  ],
+  "properties": {
+    "adapter_name": {
+      "type": "string",
+      "title": "Name",
+      "default": "",
+      "description": "Provide a unique name for this LLM adapter instance. Example: ollama-instance-1"
+    },
+    "model": {
+      "type": "string",
+      "title": "Model",
+      "default": "",
+      "description": "Provide the model name to be used. Example:llama2, llama3, mistral"
+    },
+    "base_url": {
+      "type": "string",
+      "title": "Base URL",
+      "default": "",
+      "description": "Provide the base URL where Ollama server is running."
+    },
+    "context_window": {
+      "type": "number",
+      "title": "Context window",
+      "default":3900,
+      "description": "The maximum number of context tokens for the model."
+    },
+    "request_timeout": {
+      "type": "number",
+      "title": "Request Timeout",
+      "default": 900,
+      "description": "Request timeout in seconds"
+    },
+    "json_mode": {
+      "type": "boolean",
+      "title": "JSON Mode",
+      "default": false,
+      "description": "Whether to use JSON mode for the Ollama API."
```

Comment:
Is this JSON mode supported from our prompt studio responses? Do check with @arun-venkataswamy if this is required. 
I'm curious to know if setting this to True will mess with results from prompt studio

Reply:
@chandrasekharan-zipstack When checked while running a prompt with JSON mode, it didnt cause any issues.
Also, Yes I do feel this might be usage in workflows alone, provided, we have JSON enforce type in prompt studio.
However, I would let @arun-venkataswamy confirm this.

Reply:
Let's not expose this to the end user. Let's set to false always. @chandrasekharan-zipstack is right. This will create confusion in the prompt studio